codelabs/README.md

```diff
@@ -0,0 +1,83 @@
+# Bulk Deploy Kubeflow On Codelab Test Projects
+
+This directory contains tools for automatically deploying instances
+of Kubeflow in codelab test projects.
+
+The scripts are intended to run on a kubeflow cluster.
+
+**project**: kf-codelab-admin
+**cluster**: codelab-admin
+**bucket**: gs://kf-codelab-admin
+**OAuth location**: gs://kf-codelab-admin/test-project-iap.oauth.yaml
+**GSA*: codelab-admin-user@kf-codelab-admin.iam.gserviceaccount.com
+
+
+## Permissions
+
+In order for to modify the codelab project you need to grant OWNER privileges to the GCP service
+account used by the K8s job
+
+* We use the GSA codelab-admin-user@kf-codelab-admin.iam.gserviceaccount.com
+* Which is mapped to the default-editor KSA
+
+## Instructions bulk setup of kubeflow codelab projects
+
+1. Ensure the Google Sheet containing the code lab account has a field project
+
+   * With the last batch of projects it looked like the convention was 
+
+     ```
+     USER=devstar${NUMBER}@gcplab.me
+     PROJECT=kf-test-${NUMBER}
+     ```
+
+1. Export the Google Sheet containing the code lab accounts to CSV
```

Comment:
Can you add more detailed on this step, e.g. how the CSV is formatted?

Reply:
Done.